wsmaster/che-core-api-user/src/main/java/org/eclipse/che/api/user/server/jpa/JpaUserDao.java

```diff
@@ -172,15 +172,20 @@ public UserImpl getByEmail(String email) throws NotFoundException, ServerExcepti
 
     @Override
     @Transactional
-    public Page<UserImpl> getAll(int maxItems, int skipCount) throws ServerException {
+    public Page<UserImpl> getAll(int maxItems, long skipCount) throws ServerException {
         // TODO need to ensure that 'getAll' query works with same data as 'getTotalCount'
         checkArgument(maxItems >= 0, "The number of items to return can't be negative.");
         checkArgument(skipCount >= 0, "The number of items to skip can't be negative.");
+
+        /* 'managerProvider.get().setFirstResult(int)' requires int value in its parameters
```

Comment:
i'd say that comment is useless here + i don't think that commenting all the check where page is returned may be useful either, can you please remove it ?